src/GraphQL.Tests/Types/BooleanGraphTypeTests.cs

```diff
@@ -43,5 +44,46 @@ public void coerces_string_True()
         {
             type.ParseValue("True").ShouldBe(true);
         }
+
+        [Fact]
+        public void coerces_string_1_to_true()
+        {
+            type.ParseValue("1").ShouldBe(true);
+        }
+
+        [Fact]
+        public void coerces_zero_string_to_false()
+        {
+            type.ParseValue("0").ShouldBe(false);
+        }
+
+        [Theory]
+        [InlineData("abc")]
+        [InlineData("21")]
+        public void coerces_input_to_exception(string input)
+        {
+            FormatException formatException = Assert.Throws<FormatException>(() => type.ParseValue(input));
+            Assert.Equal("String was not recognized as a valid Boolean.", formatException.Message);
+        }
+
+        [Theory]
+        [InlineData(0)]
+        [InlineData("0")]
+        [InlineData("False")]
+        [InlineData("false")]
```

Comment:
For completeness can you add an actual Boolean value?

Reply:
Done.